src/nats.h

```diff
@@ -611,6 +612,14 @@ typedef struct jsStreamConfig {
         /// @brief Sets the limits on certain options on all consumers of the
         /// stream.
         jsStreamConsumerLimits  ConsumerLimits;
+
+        /// @brief Allow the message to be sent with a time to live (TTL) value.
+        bool                    AllowMsgTTL;
+
+        /// @brief Enables and sets a duration for adding server markers for
+        /// delete, purge and max age limits.
```

Comment:
No mention of the unit here? Since the added `MsgTTL` is in milliseconds, what is this one?

Reply:
@kozlovic I'm updating the comment, but there is a potential (in-)consistency issue here. Here, in `jsStreamConfig` we already have the "prior art" of `MaxAge` which is in ns (go `time.Duration`) and we have several other places in the code where `time.Duration` is represented as nanoseconds in C, without conversions.

Yet, in `jsPubOptions` we have `MaxWait` which is in ms, so I felt compelled to make `MsgTTL` use the same units, esp. considering that the minimum allowed value is 1s.

Thoughts? Recommendation?

Reply:
I don't like the inconsistencies, but at least make sure that we are explicit in the field's comment on what the unit is. If you think millisecond is the way to go, I have no problem. It is true, that the timeouts/maxwait, etc.. are in ms.

PS: In the Go client, those fields are documented that we need a v2.11.0+ server. Should you add that too?

Reply:
Added.